tests/remove-data-test-id.spec.js

```diff
@@ -70,7 +77,6 @@ const runTests = (label, transform) => {
 
         it('throws error when attributes is empty array', () => {
           const code = '<p selenium-id={false}></p>';
-          const expectedCode = '<p></p>';
```

Comment:
I just removed un used variables from here. Maybe I can add `eslint` to the project in another PR to provide rules for future changes so that everything follows a specified format?

Reply:
I was thinking the same thing (adding eslint) ... ticket created.  My formatter is also completely opposite of this repository :D